compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_instruction_gen.bal

```diff
@@ -357,6 +359,18 @@ type InstructionGenerator object {
         self.storeToVar(binaryIns.lhsOp.variableDcl);
     }
 
+    function generateAnnotAccessIns(bir:BinaryOp binaryIns) {
+        self.loadVar(binaryIns.rhsOp1.variableDcl);
+        addBoxInsn(self.mv, binaryIns.rhsOp1.variableDcl.typeValue);
```

Comment:
may be.. might not need addBoxInsn(). just double check

Reply:
Yeah, this shouldn't be needed. Removed with cb6db58.